build.gradle.kts

```diff
@@ -37,7 +37,7 @@ configurations.all {
 }
 
 group = "com.thevoxelbox"
-version = "2.1.1-SNAPSHOT"
+version = "2.1.2-SNAPSHOT"
```

Comment:
Unneeded. If we plan on drafting a release, we'll strip the `-SNAPSHOT` suffix and adjust the version number if needed. That'll trigger a new release with said version. Appending the snapshot suffix solely differs between deploying maven artifacts to the snapshot or releases repository.

Reply:
Ok, thx for clarifications. Shouldn't we consider a release then?

Reply:
I'm more interested in rewriting the archive/deployment workflow to return to rolling releases instead of drafting releases every couple of days.
We can pull this PR and release 2.1.1 and ima address previous concerns later on.

Reply:
Makes sense, done!